cloudbuild.yaml

```diff
@@ -152,19 +154,12 @@ steps:
   - name: 'go-vol'
     path: '/go'
 
-- id: 'Test: Delete Cluster'
+# Disable ci-reap-clusters for now. TODO: refactor it to ci-reap-namespace
+- id: 'Test: Delete Open Match'
   name: 'gcr.io/$PROJECT_ID/open-match-build'
-  args: ['make', 'SHORT_SHA=${SHORT_SHA}', 'GCLOUD_EXTRA_FLAGS=--async', 'GCP_PROJECT_ID=${PROJECT_ID}', 'ci-reap-clusters', 'delete-gke-cluster']
+  args: ['make', 'SHORT_SHA=${SHORT_SHA}', 'GCLOUD_EXTRA_FLAGS=--async', 'GCP_PROJECT_ID=${PROJECT_ID}', 'ci-reap-clusters', 'delete-chart']
```

Comment:
Delete namespace?

Reply:
`delete-chart` delete the deployment and then `kubectl delete` the namespace

Reply:
ah, I missed that delete chart removed the namespace already.